platform/darwin/src/MGLFeature.mm

```diff
@@ -5,14 +5,30 @@
 #import "MGLPolygon.h"
 #import "MGLValueEvaluator.h"
 
+#import "MGLShape_Private.h"
 #import "MGLMultiPoint_Private.h"
+#import "MGLPolyline+MGLAdditions.h"
+#import "MGLPolygon+MGLAdditions.h"
+#import "NSDictionary+MGLAdditions.h"
+
+#import "NSExpression+MGLAdditions.h"
 
 #import <mbgl/util/geometry.hpp>
+#import <mapbox/geometry/feature.hpp>
+
+@interface MGLFeatureUtility: NSObject
```

Comment:
This class doesn't seem to be adding much. The lone function can be a standalone C++ function.